keyboards/rico/phoenix_project_no1/keymaps/default/keymap.c

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2023 Eric Becourt (Rico at https://github.com/mymakercorner)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+// clang-format off
+
+enum layer_names {
+    _BASE,
+    _FN,
+    _FN2,
+    _FN3
```

Comment:
```suggestion
    _FN2
```

Reply:
Hello zvecr,

It has been a long time since I resolved this issue.
Could you please validate(or not) that this issue has been fixed ?

Thanks in advance ! 

Best Regards,